ext/pdo/pdo_stmt.c

```diff
@@ -656,33 +655,16 @@ static bool make_callable_ex(pdo_stmt_t *stmt, zval *callable, zend_fcall_info *
 		}
 		return false;
 	}
-	if (is_callable_error) {
-		/* Possible error message */
-		efree(is_callable_error);
-	}
+	ZEND_ASSERT(is_callable_error == NULL);
```

Comment:
Is this really necessary? Shouldn't `zend_fcall_info_init` report failure reliably?

Reply:
The point is that there shouldn't be an error message to free if `zend_fcall_info_init` has failed.

This just adds an assertion that this indeed can never be the case.